Source/WebCore/editing/cocoa/WebContentReaderCocoa.mm

```diff
@@ -516,7 +516,7 @@ static String sanitizeMarkupWithArchive(LocalFrame& frame, Document& destination
         auto subframeMarkup = sanitizeMarkupWithArchive(frame, destinationDocument, subframeContent, MSOListQuirks::Disabled, canShowMIMETypeAsHTML);
 
         CString utf8 = subframeMarkup.utf8();
-        auto blob = Blob::create(&destinationDocument, Vector { utf8.dataAsUInt8Ptr(), utf8.length() }, type);
+        auto blob = Blob::create(&destinationDocument, Vector<uint8_t> { utf8.bytes() }, type);
```

Comment:
Yes, will make it work.

Reply:
`Vector(utf8.bytes())` actually does work. `Vector { utf8.bytes() }` doesn't because it is trying to constructor a `Vector<std::span<const uint8_t>>`.